endpoints/put_test.go

```diff
@@ -593,10 +598,12 @@ func TestBadPayloadSizePutError(t *testing.T) {
 	m := metricstest.CreateMockMetrics()
 	router.POST("/cache", NewPutHandler(backend, m, 10, true))
 
-	_, httpTestRecorder := doMockPut(t, router, reqBody)
+	putResponse := doPut(t, router, reqBody)
+
+	// Assert expected response
+	assert.Equal(t, http.StatusBadRequest, putResponse.Code, "doMockPut should have failed when trying to store elements in sizeCappedBackend")
```

Comment:
Nitpick: the string at the end refers to `doMockPut` but it should now be `doPut`